pages/common/speed-test.md

```diff
@@ -0,0 +1,20 @@
+# speed-test
+
+> Test your internet connection speed and ping using speedtest.net from the CLI.
+> More information: <https://github.com/sindresorhus/speed-test/>.
+
+- Test your internet connection speed and ping:
```

Comment:
```suggestion
- Test your internet connection speed and ping speed:
```

Reply:
What about making it "Test your internet connection and ping speed" instead, which is shorter?

Reply:
Suggestion updated.